tools/check_format.py

```diff
@@ -164,13 +164,24 @@
 # yapf: enable
 
 
-# Map a line transformation function across each line of a file.
-# .bak temporaries.
-def replaceLines(path, line_xform):
+# Map a line transformation function across each line of a file,
```

Comment:
Thanks this is a nice cleanup. @jmarantz do you mind taking a quick look at just this part of the change?

Reply:
Thank @sunjayBhatia :) Yes, it's much cleaner now.